en/12_Project_Governance/07_Supported_Modules.md

```diff
@@ -7,7 +7,8 @@ description: Modules which are commercially supported by Silverstripe
 
 Silverstripe CMS ships modules that receive commercial support. Commercially supported modules receive regular updates to work with the latest Silverstripe CMS release. Their [public API](/project_governance/public_api/) conforms to [semantic versioning](https://semver.org/). They are covered by:
 
-- our [security release process](/contributing/managing_security_issues) and
+- our [security release process](/contributing/managing_security_issues)
+- our [minor release policy](minor_release_policy)
 - our [major release policy](major_release_policy).
```

Comment:
This is an unrelated change - I noticed we don't call out the minor release policy here but we should.